src/core/EdFi.DataManagementService.Core/Middleware/CoreLoggingMiddleware.cs

```diff
@@ -27,18 +27,7 @@ public async Task Execute(PipelineContext context, Func<Task> next)
         {
             _logger.LogError(ex, "Unknown Error - {TraceId}", context.FrontendRequest.TraceId);
 
-            // Replace the frontend response (if any) with a 500 error
-            context.FrontendResponse = new(
-                StatusCode: 500,
-                Body: JsonSerializer.Serialize(
-                    new
-                    {
-                        message = "The server encountered an unexpected condition that prevented it from fulfilling the request.",
-                        traceId = context.FrontendRequest.TraceId
-                    }
-                ),
-                Headers: []
-            );
+            FailureResponse.GenerateFrontendErrorResponse(context, ex.Message, 400);
```

Comment:
Undo this. The purpose of CoreLoggingMiddleware's exception handling is to throw 500's, like it says in the comments.